build/__snapshots__/features-meta.json

```diff
@@ -748,6 +748,12 @@
 		"description": "Retries downloading images that failed downloading due to GitHub limited proxying.",
 		"screenshot": "https://user-images.githubusercontent.com/14858959/64068746-21991100-cc45-11e9-844e-827f5ac9b51e.png"
 	},
+	{
+		"id": "render-conventional-commit-types",
+		"description": "Renders conventional commit types as labels prefixing the commit message.",
+		"css": true,
+		"screenshot": "https://github.com/user-attachments/assets/2a5ae00e-f440-49ef-b810-aa9a5a2ac269"
```

Comment:
Where did you upload this? The URL looks fine but maybe we need to update the feature checker. 

What I do is drop the files in the PR comments and take the URL as is. If that’s what you did, we’ll have to update the feature checker to also accept this format.

Reply:
I dropped it in the comment text box. Also tried in the code editor.

Reply:
I think it actually works now. CI is only failing because the snapshots are not up to date.